docs/source/_topic_map.yml

```diff
@@ -59,6 +57,17 @@ Topics:
         File: writing-docs
       - Name: Releasing Minishift
         File: releasing
+  - Name: Troubleshooting Minishift
+    Dir: troubleshooting
+    Topics:
+      - Name: Overview
+        File: index
+      - Name: Troubleshooting General
```

Comment:
I move this last and I'd prefer the term 'Misc' or 'Miscellaneous'. IMO general does not quite have the right meaning.